.github/workflows/ci.yml

```diff
@@ -73,11 +73,11 @@ jobs:
                           --build-arg COMPILER=${{ matrix.compiler }}
                           --build-arg DEPS_JOBS=8 --build-arg DAOS_JAVA_BUILD=no
                           --build-arg BASE_DISTRO=fedora:34
-#    - name: Build DAOS Java client in Docker image
-#      run: docker build . -f utils/docker/Dockerfile.centos.8
-#                          --build-arg COMPILER=${{ matrix.compiler }}
-#                          --build-arg DEPS_JOBS=8
-#                          --build-arg BASE_DISTRO=fedora:34
+    - name: Build DAOS Java client in Docker image
+      run: docker build . -f utils/docker/Dockerfile.centos.8
+                          --build-arg COMPILER=${{ matrix.compiler }}
+                          --build-arg DEPS_JOBS=8
+                          --build-arg BASE_DISTRO=fedora:34
```

Comment:
FYI - you can test this code locally by running the command from here:

docker build . -f utils/docker/Dockerfile.centos.8 --build-arg BASE_DISTRO=fedora:34

If you don't have a docker server setup I can let you access one of mine on the wolf cluster.

Reply:
that would be great. please share me your docker server in wolf. thanks.